glide-core/redis-rs/redis/src/cluster_routing.rs

```diff
@@ -259,7 +352,11 @@ pub(crate) fn combine_and_sort_array_results<'a>(
         }),
         Value::Nil,
     );
-    assert_eq!(values.len(), sorting_order.len());
+    assert_eq!(
```

Comment:
are we asserting in production code? we should probably raise an error instead.
If you want debug asserts, please use `debug_assert_eq!`

Reply:
changed